pom.xml

```diff
@@ -122,7 +122,7 @@
         <xpp3.servicemix.version>1.1.4c_7</xpp3.servicemix.version>
         <kxml2.servicemix.version>2.3.0_3</kxml2.servicemix.version>
         <!-- double-check downstream projects before changing jackson version -->
-        <fasterxml.jackson.version>2.9.8</fasterxml.jackson.version> <!-- To match cxf-jackson (from cxf-jaxrs) -->
+        <fasterxml.jackson.version>2.10.0.pr2</fasterxml.jackson.version> <!-- 2.9.9 matches cxf-jackson (from cxf-jaxrs), but 2.10 has better yaml support -->
```

Comment:
Why `2.10.0.pr2`? Don't know what "pr" means, but it sounds like a release candidate.

`2.10.0` or `2.10.1` looks newer than `2.10.pr2` - https://search.maven.org/artifact/com.fasterxml.jackson.dataformat/jackson-dataformat-xml